src/newpm/passes.jl

```diff
@@ -0,0 +1,1349 @@
+export NewPMPass, NewPMLLVMPass
```

Comment:
This is quite some code to maintain. Can't we use some metaprogramming to start from a more compact (and thus easier to maintain) listing, just like we did for the current passes?

Reply:
I've done this, though the only downside I can see is that coverage data will probably be a little less useful for the passes (it shouldn't matter though since we test adding every pass to the appropriate pass manager).

Reply:
Awesome; thanks!